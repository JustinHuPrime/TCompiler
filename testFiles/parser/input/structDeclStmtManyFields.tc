module foo;

void bar() {
  struct s {
    int x, y;
    double r;
  };
}