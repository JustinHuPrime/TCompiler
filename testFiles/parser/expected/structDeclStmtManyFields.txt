testFiles/parser/structDeclStmtManyFields.tc (code):
FILE(1, 1, STAB(ENTRY(bar, FUNCTION(testFiles/parser/structDeclStmtManyFields.tc, 3, 1, void()))), MODULE(1, 1, ID(1, 8, foo, REFERENCES())), FUNDEFN(3, 1, KEYWORDTYPE(3, 1, void), ID(3, 6, bar, REFERENCES(testFiles/parser/structDeclStmtManyFields.tc, 3, 1)), STAB(), COMPOUNDSTMT(3, 12, STAB(ENTRY(s, STRUCT(testFiles/parser/structDeclStmtManyFields.tc, 4, 3, FIELD(int, x), FIELD(int, y), FIELD(double, r)))), STRUCTDECL(4, 3, ID(4, 10, s, REFERENCES(testFiles/parser/structDeclStmtManyFields.tc, 4, 3)), VARDECL(5, 5, KEYWORDTYPE(5, 5, int), ID(5, 9, x, REFERENCES()), ID(5, 12, y, REFERENCES())), VARDECL(6, 5, KEYWORDTYPE(6, 5, double), ID(6, 12, r, REFERENCES()))))))
